frontend/public/components/secrets/create-secret.tsx

```diff
@@ -373,12 +394,14 @@ export class ImageSecretForm extends React.Component<ImageSecretFormProps, Image
             onChange={this.onDataChanged}
             stringData={data}
             onDisable={this.onFormDisable}
+            t={this.props.t}
```

Comment:
Don't pass `t` as a prop. Use `useTranslation` or `withTranslation` in the component instead.